XCUITests/CronTabsPerfTests.swift

```diff
@@ -0,0 +1,243 @@
+
+    override func setUp() {
+        // Test name looks like: "[Class testFunc]", parse out function name
+        let parts = name.replacingOccurrences(of: "]", with: "").split(separator: " ")
+        let functionName = String(parts[1])
+        let archiveName = fixtures[functionName]
+
+        // defaults
+        launchArguments = [LaunchArguments.PerformanceTest, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew, LaunchArguments.SkipETPCoverSheet]
+
+        // append specific load profiles to LaunchArguments
+        if fixtures.keys.contains(functionName) {
+            launchArguments.append(LaunchArguments.LoadTabsStateArchive + archiveName!)
+        }
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    // TODO: 1 perf test per tabsStateArchive of size:
+    // 1, 10, 20, 40, 80, 160, 320, etc.(tab count)
+    
+    // baseline
+//    func testPerfTabs1() {
+//        if #available(iOS 13.0, *) {
+//            measure(metrics: [
+//                XCTClockMetric(), // to measure timeClock Mon
+//                XCTCPUMetric(), // to measure cpu cycles
+//                XCTStorageMetric(), // to measure storage consuming
+//                XCTMemoryMetric()]) {
+//                app.launch()
+//            }
+//        }
+//    }
+    
+    func testPerfTabs10() {
+        if #available(iOS 13.0, *) {
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                app.launch()
+                // activity measurement here
+            }
+        }
+    }
+
+    func testPerfTabs20startup() {
+        if #available(iOS 13.0, *) {
+
+            measure(metrics: [
+                XCTMemoryMetric(),
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+            }
+        }
+    }
+
+    func testPerfTabs1280startup() {
+        if #available(iOS 13.0, *) {
+
+            measure(metrics: [
+                XCTMemoryMetric(),
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+            }
+        }
+    }
+    func testPerfTabs40() {
```

Comment:
I thought we were going to have only the start up test for 10 and 1280 tabs (or the ones we decide in the extremes). I would say to keep these tests but commented so that we don't have to upload to the repo all the corresponding archive files... We can upload those archives to a shared drive in case they are needed in the future. I know they are not very heavy, but add too many extra files... I think it may be a good idea to add a document to the [Docs folder](https://github.com/mozilla-mobile/firefox-ios/tree/main/Docs) explaining about these tests, how to run them and we could link the gdrive in there... Of course, there is no need to do that in this PR, a follow up would be fine